stages.yml

```diff
@@ -66,9 +69,9 @@ stages:
 
   - template: documentation.yml
     parameters:
-      image: ${{ parameters.settings.linux.image }}
-      conda_package_root: ${{ parameters.settings.linux.conda_package_root }}
-      conda_env: ${{ parameters.config.linux.conda_env }}
+      image: ${{ parameters.settings.noarch.image }}
+      conda_package_root: ${{ parameters.settings.noarch.conda_package_root }}
+      conda_env: ${{ parameters.config.noarch.conda_env }}
```

Comment:
Need a fix here to make it general. Should check that we have `noarch` in `config` before choosing which platform to use for documentation build. Changes here would not work for `scippneutron` for example